lib/unit/dialog/dialog.dart

```diff
@@ -0,0 +1,24 @@
+
+
+import 'package:flutter/material.dart';
+
+
+void showConfirmDialog(BuildContext context, String? message) {
+  showDialog(
+    context: context,
+    barrierDismissible: false,
+    builder: (BuildContext dialogContext) {
+      return AlertDialog(
+        title: Text(message!),
```

Comment:
```suggestion
        title: Text(message),
```
にしましょう！
nullをアンラップすることはおすすめしないです。。。